src/Main/Main.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * The file that defines the main start class.
+ *
+ * A class definition that includes attributes and functions used across both the
+ * theme-facing side of the site and the admin area.
+ *
+ * @package EightshiftForms\Main
+ */
+
+declare(strict_types=1);
+
+namespace EightshiftForms\Main;
+
+use EightshiftLibs\Main\AbstractMain;
+use EightshiftForms\Cache;
+use EightshiftForms\Captcha;
+use EightshiftForms\Rest;
+use EightshiftForms\Integrations;
+use EightshiftFormsTests\Mocks;
+
+/**
+ * The main start class.
+ *
+ * This is used to define admin-specific hooks, and
+ * theme-facing site hooks.
+ *
+ * Also maintains the unique identifier of this theme as well as the current
+ * version of the theme.
+ */
+class Main extends AbstractMain
+{
+
+	/**
+	 * Set this to true if you need dependency injection in tests.
+	 *
+	 * @var boolean
+	 */
+	private $isTest = false;
+
+	/**
+	 * Register the project with the WordPress system.
+	 *
+	 * The registerService method will call the register() method in every service class,
+	 * which holds the actions and filters - effectively replacing the need to manually add
+	 * them in one place.
+	 *
+	 * @return void
+	 */
+	public function register(): void
+	{
+		\add_action('after_setup_theme', [$this, 'registerServices']);
```

Comment:
Are you intentionally registering all the services this late? Usually, plugins are hooking as early as `plugins_loaded` 🤔

Reply:
Dunno, that was copied from Main in Libs https://github.com/infinum/eightshift-libs/blob/develop/src/Main/MainExample.php 🤷‍♂️

Reply:
Yeah, but that is because we target libs usage in themes first. This should be changed in the plugin boilerplate to load things in the `plugins_loaded`.

Reply:
Yeah makes sense, changed!